### PR TITLE
Updating Realtime Example to reflect more options.

### PR DESCRIPTION
**Pull request by mjrobison**

### CODE REVIEW

**Comment by murphycrosby** [comment]
Looks good 🤣

File: examples/python/realtime-client.py [hunk 1]
```diff
@@ -105,10 +110,18 @@ def get_records(bearer_token_, base_url_, shard_iterator_, limit_):
...
-                        choices=['TRIM_HORIZON', 'LATEST'],
+                        choices=['TRIM_HORIZON', 'LATEST', 'AT_SEQUENCE', 'AFTER_SEQUENCE', 'AT_TIMESTAMP'],
                         )
+    parser.add_argument('-u', '--unixTime', help='When using AT_TIMESTAMP provide a time in miliseconds')
+    parser.add_argument('-n', '--sequenceNumber', help='When using AT_SEQUENCE or AFTER_SEQUENCE provide a starting sequence number')
     args = parser.parse_args()
 
+    if args.shardType == 'AT_TIMESTAMP' and args.unixTime is None:
+        raise Exception("Unix formatted starting time is requried to use the AT_TIMESTAMP iterator")
+    if args.shardType in ['AT_SEQUENCE', 'AFTER_SEQUENCE'] and args.unixTime is None:
```
Comment (NRBriggs) [inline examples/python/realtime-client.py]: Small typo on line 120: `requried`

Line 121: `args.unixTime` might ought to be `args.sequenceNumber`

Comment (mjrobison) [reply examples/python/realtime-client.py]: Can't rely on me to get through one without missing something.  🤦

**Review by NRBriggs — APPROVED** [review]
Looks good!

**Pull request merged**